packages/mux-player/src/index.ts

```diff
@@ -733,20 +735,41 @@ class MuxPlayerElement extends VideoApiElement {
   }
 
   /**
-   * Return the storyboard URL when a playback ID is provided,
+   * Return the storyboard-src attribute URL
+   */
+  get storyboardSrc() {
+    return this.getAttribute(PlayerAttributes.STORYBOARD_SRC) ?? '';
+  }
+
+  /**
+   * Set the storyboard-src attribute URL
+   */
+  set storyboardSrc(src: string) {
+    if (!src) {
+      this.removeAttribute(PlayerAttributes.STORYBOARD_SRC);
+    } else {
+      this.setAttribute(PlayerAttributes.STORYBOARD_SRC, src);
+    }
+  }
+  /**
+   * Return the storyboard URL when a playback ID or storyboard-src is provided,
    * we aren't an audio player and the stream-type isn't live.
    */
   get storyboard() {
     if (
-      this.playbackId &&
       !this.audio &&
       (!this.streamType ||
         ![StreamTypes.LIVE, StreamTypes.LL_LIVE, StreamTypes.DVR, StreamTypes.LL_DVR].includes(this.streamType as any))
     ) {
-      return getStoryboardURLFromPlaybackId(this.playbackId, {
-        domain: this.customDomain,
-        token: this.tokens.storyboard,
-      });
+      // only infer from playbackId if not set on storyboardSrc
+      if (!this.storyboardSrc && this.playbackId) {
+        return getStoryboardURLFromPlaybackId(this.playbackId, {
+          domain: this.customDomain,
+          token: this.tokens.storyboard,
+        });
+      } else {
+        return this.storyboardSrc;
```

Comment:
question(blocking): Should we still respect the `storyboard-token`/`tokens.storyboard` here if it's present? If not, this may be confusing from a devex perspective.

Reply:
Related, should this just always return the set storyboardSrc if available? Regardless of whether playbackId is set or not.

Reply:
I agree that the token, if present, should be respected; thanks for that callout. 

As for only returning `this.storyboardSrc` when `this.playbackId` is set, I'm not so sure. Although it's undocumented, someone can set `<mux-player src="..." storyboard-src="..." />`, IE never setting `playbackId`.

I guess I wonder why _should_ `storyboardSrc` be tied to `playbackId` being present?

Reply:
> I guess I wonder why should storyboardSrc be tied to playbackId being present?

Your approach here makes sense imo and is consistent with the way we handle `poster`. Resolving (though we can re-open discussion if others have concerns).